classes/Context.php

```diff
@@ -347,6 +348,12 @@ public function getTranslator()
             return $this->translator;
         }
 
+        $sfContainer = SymfonyContainer::getInstance();
+        if (!is_null($sfContainer)) {
+            $this->translator = $sfContainer->get('translator');
```

Comment:
I juste remove $sfContainer->getContainer because it was already container hihi :smile: